<launch>
  <node pkg="roscpp_morai" type="pp_1026" name="pp_1026" output="screen">
    <param name="ref_file"  value="/root/ws/src/roscpp_morai/map/ref.txt"/>
    <param name="path_file" value="/root/ws/src/roscpp_morai/map/Path.txt"/>
    <remap from="/gps"       to="/gps"/>        <!-- 실제 토픽명 맞추기 -->
    <remap from="/imu"  to="/imu"/>   <!-- 실제 토픽명 맞추기 -->
    <remap from="/ego_topic" to="/Ego_topic"/>  <!-- 실제 토픽명 맞추기 -->
    <remap from="/ctrl_cmd"  to="/ctrl_cmd"/>
    <remap from="/local_path" to="/local_path"/>
  </node>
</launch>
